data/curated/enmoma/saving.R

```diff
@@ -0,0 +1,9 @@
+source("data/curated/enmoma/cleaning.R")
```

Comment:
When you run this step, it should add the `ttsave` function to your environment. Please try to use this and note any errors. The save step also creates the data dictionary file, and does some checks.